module4/projects/mod4Project/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Mod 4 Project
+---
+
+## Project Objectives
+* Practice development in a group environment
+* Work toward a self-determined goal
+* Discuss your project in professional, technical terms
+
+### Overview
+
+The goal of this project is to create a successful web application from a student-led project idea.  your team will create an app that will solve a real world problem.
```

Comment:
I would also add that they will be working in groups of 3-4 and the project will take place over weeks 5 and 6.